runtime/src/asset.rs

```diff
@@ -1612,1586 +1621,3 @@ impl<T: Trait> Module<T> {
         Ok(())
     }
 }
```

Comment:
We need to update the transfer stats on all Issue/mint and redeem functions as well. See all the places where `_update_checkpoint` is being called. Infact, we can maybe merge the status updation and checkpoints updation. I wonder if storing checkpoints in the stats module is a good idea.

Reply:
Done. Now updated balance parameters (from & to) are optional to cover _redeem_ and _mint_ operations.